Woof/Woof/Model/Walking/WalkingStatus.swift

```diff
@@ -0,0 +1,43 @@
+import Foundation
+
+/**
+ The possible walking status values for a `Walking`.
+ Conforms to the `Codable` protocol.
+ */
+enum WalkingStatus: Codable {
+    /// The walking is available for booking.
+    case available
+
+    /// The walking is pending approval from the pet sitter.
+    case pending
+
+    /// The walking is booked by owner.
```

Comment:
I think it's more correct and consistent:
```suggestion
    /// The walking has been booked by owner.
```